classes/01class/exercises/c01-git01/README.md

```diff
@@ -1,12 +1,18 @@
 ## Merging and Rebasing (c01-git01)
 
-1. Access https://learngitbranching.js.org/ (TIP: if you want to start over, just type `reset`)
-2. Merging exercise:
+1. Access https://learngitbranching.js.org/ (**TIP**: if you want to start over, just type `reset`. To dismiss tutorial screens, type `Esc`)
+2. **c01-git01-1**: Merging exercise:
    1. type `level intro3`
-   2. Submission: Include your commands on a file **c01-git01-1.txt**
-3. Rebasing exercise:
+3. **c01-git01-2**: Rebasing exercise:
    1. type `level intro4`
-   2. Submission: Include your commands on a file **c01-git01-2.txt**
-4. Revert and reset: 
+4. **c01-git01-3**: Revert and reset:
    1. type `level rampup4`
-   2. Submission: Include your commands on a file **c01-git01-3.txt**
\ No newline at end of file
+
+## Submit a PR with the following files
+
+> Remember to follow the instructions on [how to submit a PR here](/README.md#exercises)s
+
+- **README.md**: copy from file [ANSWER.md](ANSWER.md), containing links to the following files with your answers:
+  - **c01-git01-1.md**: Commands typed to solve the "Merging" exercise
+  - **c01-git01-2.md**: Commands typed to solve the "Rebasing" exercise
+  - **c01-git01-3.md**: Commands typed to solve the "Revert and reset" exercise
```

Comment:
If we're not providing a .md template for them, maybe we should ask them to include the commands inside the ANSWER file, with a pre templated file. Otherwise we can just keep .txt file as it won't matter much without formatting.

Reply:
They will insert git commands in the answer and it is better to have this as md then txt in my opinion (they can get creative and format the response). Unless we are thinking in automating correction and want raw answers

Reply:
Got it.. then should we have a single README.md file withh all answers, instead of having a README file that only serve as a index?